src/components/print/PrintDirective.js

```diff
@@ -653,7 +653,7 @@ goog.require('ga_urlutils_service');
                 'graphicYOffset': -30
               }, '2': { // Style for measure tooltip
                 'externalGraphic':
-                  gaUrlUtils.nonCloudFrontUrl($scope.options.bubbleUrl),
+                  gaUrlUtils.unProxifyUrl($scope.options.bubbleUrl),
```

Comment:
`bubbleUrl` and `markerUrl` doesn't use proxy I think, since they are geoadmin images